lib/rubocop/cop/rspec/single_argument_message_chain.rb

```diff
@@ -67,11 +67,9 @@ def single_element_array?(node)
         end
 
         def autocorrect_hash_arg(corrector, arg)
-          key, value = *arg.children.first
-
-          corrector.replace(arg, key_to_arg(key))
+          corrector.replace(arg, key_to_arg(arg.keys.first))
```

Comment:
how about
```ruby
pair = arg.pairs.first
```
and then using `key_to_arg(pair.key)` and `pair.value.source` respectively ?

Reply:
Yep, that’s even better. Thank you.